RestructuredText/reStructuredText.sublime-syntax

```diff
@@ -9,44 +9,42 @@ file_extensions:
 scope: text.restructuredtext
 contexts:
   main:
-    - match: '^([ \t]*)(?=\S)'
+    - match: '^(?=(\s*)\S)'
       push:
         - meta_content_scope: meta.paragraph.restructuredtext
-        - match: ^(?!\1(?=\S))
+        - match: '^(?!\1\s+)(?=\s*\S+)'
           pop: true
         - include: inline
   inline:
-    - match: '^([ \t]*)((\.\.)\sraw(::)) html'
+    - match: '^(\s*)((\.\.)\sraw(::)) html'
       comment: directives.html
       captures:
         2: meta.directive.restructuredtext
         3: punctuation.definition.directive.restructuredtext
         4: punctuation.separator.key-value.restructuredtext
       push:
-        - match: '^(?!\1[ \t])'
-          captures:
-            2: meta.directive.restructuredtext
-            3: punctuation.definition.directive.restructuredtext
-            4: punctuation.separator.key-value.restructuredtext
+        - match: '^(?!\1\s+)(?=\s*\S+)'
           pop: true
+        - meta_content_scope: text.html.basic
         - include: scope:text.html.basic
-    - match: '(\.\.)\s[A-z][A-z0-9-_]+(::)\s*$'
+    - match: '^(\s*)((\.\.)\s[A-z][A-z0-9\-_]+(::))\s*'
       comment: directives
-      scope: meta.other.directive.restructuredtext
       captures:
-        1: punctuation.definition.directive.restructuredtext
-        2: punctuation.separator.key-value.restructuredtext
-    - match: '^([ \t]*).*?((::))'
+        2: meta.other.directive.restructuredtext
+        3: punctuation.definition.directive.restructuredtext
+        4: punctuation.separator.key-value.restructuredtext
```

Comment:
These aren't really key-value pairs. The colons rather serve the purpose of starting the directive's body.

Reply:
This is just continuing with what was already there - I'd prefer to leave scope names out of scope for this review, since I didn't change any of them.

Reply:
I only looked at the diff and noticed that you added this scope, but I see that it is used elsewhere already. I agree that this isn't the place to change them.